SDNExpress/scripts/FabricConfig.example.psd1

```diff
@@ -40,6 +40,19 @@
             #The HostUsername must contain the Domain name in the format DOMAIN\User
             HostUsername = 'CONTOSO\al'                
             HostPassword = 'MySuperS3cretP4ssword'     
+            
+            #iDNS IP address and resource ID. The creds are same as the domain creds
+            iDNSCredentialResourceId = "c6abefg6-44fb-45f6-89ec-5ebd890a144f"            
+            iDNSAddress= '10.60.34.9'                                      #Example: "10.0.0.7"
+            iDNSZoneName = 'contoso.local'                                    #Example: "contoso.local"
+            DnsPort=53                                                        #Standard DNS port. DO NOT CHANGE            
+            DnsIPAddress="169.254.169.254"                                    #Fixed IP address understood by DNS proxy. DO NOT CHANGE
+            DnsProxyServiceName="DnsProxy"                                    #Name of the service. DO NOT CHANGE
+            iDNSMacAddress = 'AA-BB-CC-AA-BB-CC'                                  #Example "AA-BB-CC-AA-BB-CC"
+
+            #iDNS Proxy Registry Hives
+            iDnsVfpPath = "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\NcHostAgent\Parameters\Plugins\Vnet\InfraServices\DnsProxyService"                                                                                          
```

Comment:
I don't think we should be exposing a registry path in the configuration file. This doesn't seem like something which is actually configurable? If this is a fixed path and it is needed by SDNExpress.ps1 then it would be better to have it within SDNExpress.ps1 itself (looks like only one Configuration called "ConfigureIDnsProxy" needs this registry path).